app/src/store/playground/types.ts

```diff
@@ -101,9 +81,9 @@ export type ModelConfig = {
 /**
  * The type of a tool in the playground
  */
-export type Tool = {
+export type OpenAITool = {
```

Comment:
this is just being more honest in preparation for the conversion from one tool schema to another